### PR TITLE
[PODS-7076: Create Connector and Controller aligning with stubs response for Event 1 POST API 'Create Compiled Event 1 Report' (1827)]

### PR DESCRIPTION
**Pull request by ngmjohnstone**
Continued in 7145

### CODE REVIEW

File: app/connectors/EventReportConnector.scala [hunk 1]
```diff
@@ -58,6 +61,19 @@ class EventReportConnectorImpl @Inject()(
     }
   }
 
+  override def compileEventOneReport(pstr: String, data: JsValue)
+                                        (implicit headerCarrier: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] = {
+    val compileEvent1ReportUrl = config.compileEvent1ReportUrl.format(pstr)
```
Comment (MazinGhafadgi) [inline app/connectors/EventReportConnector.scala:66]: any reason why the log is set to warning?

File: app/controllers/EventReportController.scala [hunk 2]
```diff
@@ -64,6 +64,23 @@ class EventReportController @Inject()(
       }
   }
 
+  def compileEventOneReport: Action[AnyContent] = Action.async {
+    implicit request =>
+      post { (pstr, userAnswersJson) =>
+        logger.debug(message = s"[Compile Event 1 Report: Incoming-Payload]$userAnswersJson")
```
Comment (MazinGhafadgi) [inline app/controllers/EventReportController.scala:70]: better if you remove the commented code until it is ready.

Comment (ngmjohnstone) [reply app/controllers/EventReportController.scala:70]: This was left unimplemented as validation was part of 7145, now implemented: https://github.com/hmrc/pension-scheme-event-reporting/pull/11/files#diff-1e7681c64f3d16a12b604f6fad9a7e98cb624f0cd37af940a018028ee6480893

File: app/connectors/EventReportConnector.scala (hunk 1 above)
Comment (ngmjohnstone) [reply app/connectors/EventReportConnector.scala:66]: Log set to warning matches previous implementation used in AFT

File: app/controllers/EventReportController.scala (hunk 2 above)
Comment (MazinGhafadgi) [reply app/controllers/EventReportController.scala:70]: I though we are doing TDD ;). writing the test before implementation , I will resolve it it for now, but please in future we should be restricted within the scope of the ticket. thanks

File: app/connectors/EventReportConnector.scala (hunk 1 above)
Comment (MazinGhafadgi) [reply app/connectors/EventReportConnector.scala:66]: better if we set it to Info or debug, not sure why is set to warn in AFT!

**Comment by MazinGhafadgi** [comment]
This PR already merged with PR 11 therefore we are closing it.

**Pull request merged**